lib/oli_web/live/products/payments/create_codes.ex

```diff
@@ -7,32 +7,37 @@ defmodule OliWeb.Products.Payments.CreateCodes do
   prop change, :event, required: true
   prop disabled, :boolean, required: true
   data something, :any, default: true
+  prop enabled_download, :boolean, required: true
 
   def render(assigns) do
     ~F"""
-    <div>
+    <div class="d-flex justify-content-between align-items-center">
       <div class="form-inline">
 
         <p>Download a new batch of payment codes:</p>
-        <input class="ml-2 form-control form-control-sm" disabled={@disabled} type="number" value={@count} style="width: 90px;" :on-blur={@change} :on-keyup={@change}/>
+        <input class="ml-2 form-control form-control-sm" disabled={@disabled} type="number" value={@count} style="width: 90px;" :on-blur={@change} :on-keyup={@change} :on-click={@change}/>
```

Comment:
I think it is necessary because if I don't call 'on-click' in the numerical input it doesn't change the state of the property I use to verify if the download button has to be enabled or not. 
Since I need to disable the download button on every interaction, I also need to manage to disable it when the numerical input changes by clicking on the up and down arrows.

Reply:
Since there is no "on-change" event that contemplates all the interactions with the component, I solved it by using the focus and blur events of the component.